tests/cases/layout/box_precentages.60

```diff
@@ -20,7 +20,6 @@ TestCase := Rectangle {
             rect1 := Rectangle {
                 background: red;
                 width: 10%;
-                height: 20%;
```

Comment:
Just curious, what happened here?

Reply:
Before, the red box was smaller than the blue box.  
(see https://sixtyfps.io/editor/preview.html?load_url=https://raw.githubusercontent.com/sixtyfpsui/sixtyfps/master/tests/cases/layout/box_precentages.60 )
But i think that was wrong.

Reply:
Ok :). Ah, and I see that that still works, in case somebody wants that behaviour.